README.adoc

```diff
@@ -198,16 +208,142 @@ curl http://localhost:9080/hello
 ```
 endif::[]
 
-=== Tearing down the Docker container
+After you are finished checking out the application, stop your container by running the following command:
+
+[role='command']
+```
+docker stop springBootContainer
+```
+
+ifndef::cloud-hosted[]
+== Optional: Faster container startup with InstantOn
+
+Open Liberty InstantOn provides fast startup times for MicroProfile and Jakarta EE applications. In this section, you’ll learn how to build an InstantOn image that provides fast startup times. Your application can start in milliseconds, without compromising on throughput, memory, development-production parity, or Java language features.
```

Comment:
```suggestion
Liberty InstantOn provides fast startup times for MicroProfile and Jakarta EE applications. In this section, you’ll learn how to build an InstantOn image that provides fast startup times. Your application can start in milliseconds, without compromising on throughput, memory, development-production parity, or Java language features.
```

Think we should just use "Liberty InstantOn" whichever Liberty we're talking about. Otherwise, it gets confusing about what's the difference between Liberty InstantOn, Open Liberty InstantOn, Semeru InstantOn, InstantOn...